cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -707,24 +707,29 @@ mod test {
         Ok(yp.ast())
     }
 
-    fn rule(n: &str) -> Symbol {
-        Symbol::Rule(n.to_string())
+    fn rule(n: &str, span: Span) -> Symbol {
```

Comment:
AFAICS we always call this with an empty span, so maybe remove the `span: Span` argument and document that the rule it creates has an empty span? It'll save space below!

Reply:
Since it is used in it seems at least one place how about i add `fn rule_span(n, span)` and leave `rule` with an empty span?
These are just private functions in the test module IIRC

Reply:
Good idea!